### PR TITLE
WW ewk corrections

### PR DESCRIPTION
**Pull request by amassiro**
Thanks to @SenneVanPutte  for spotting this.

Some comments and the origin of these corrections has been added to the code, for future reference.

A fix on the lepton pairing has been added. It was a left-over from Run 1 latino ntuples, where all objects were "pt ordered", while it's not the case here.

### CODE REVIEW

**Comment by uajanssen** [comment]
OK, we should merge before v7 I guess

**Comment by uajanssen** [comment]
and did you review the list of samples on which we apply ?

**Comment by amassiro** [comment]
Let us check also for the WW>lvjj case ... if we manage to apply it too

**Comment by amassiro** [comment]
@SenneVanPutte @valsdav : could you give it a try for the semileptonic WW>lvjj case?

The current code, in case 2 leptons are NOT found, look for two quarks and chooses the pair with invariant mass closest to the W-mass.

It works up to 4 additional quarks (meaning 2 from W + 2 from ISR/FSR).

The code can be improved and generalized.

File: NanoGardener/python/modules/qq2vvEWKcorrectionsWeightProducer.py [hunk 1]
```diff
@@ -235,8 +459,8 @@ def analyze(self, event):
           temp_etaq2 = -1
           temp_phiq1 = -1
           temp_phiq2 = -1
-          temp_idlq1 = -1
-          temp_idlq2 = -1
+          temp_idq1 = -1
```
Comment (amassiro) [inline NanoGardener/python/modules/qq2vvEWKcorrectionsWeightProducer.py:462]: ah ... and this I think was a bug left there before ... although it had no effect ... python magic assignment

**Comment by amassiro** [comment]
After lookig carefully at the referenced paper arXiv:1401.3964,
in order to have the "t" Mandelstam variable defined in the standard way for qq>WW
we check the charge of q1 and W1, such that charge (q1) has the same sign of the charge (W1)
https://en.wikipedia.org/wiki/Mandelstam_variables

**Comment by amassiro** [comment]
> and did you review the list of samples on which we apply ?

Done and updated

**Pull request merged**